test/evergreen.yml

```diff
@@ -2560,7 +2582,7 @@ buildvariants:
     - name: linux-directio
       distros: ubuntu1804-build
     - name: syscall-linux
-    - name: make-check-asan-test
+    - name: make-check-test
```

Comment:
Why adding `make-check-test` here? It should have been covered by those splitted tasks by line 2577.

Reply:
Well I was simply making sure we had that coverage, happy to remove it.